docs/azure_arc_jumpstart/azure_arc_data/day2/aks/aks_mssql_dr/_index.md

```diff
@@ -0,0 +1,346 @@
+
+  ![Screenshot showing the PowerShell logon script run](./23.png)
+
+  ![Screenshot showing the PowerShell logon script run](./24.png)
+
+  ![Screenshot showing the PowerShell logon script run](./25.png)
+
+  ![Screenshot showing the PowerShell logon script run](./26.png)
+
+  ![Screenshot showing the PowerShell logon script run](./27.png)
+
+  ![Screenshot showing the PowerShell logon script run](./28.png)
+
+  ![Screenshot showing the PowerShell logon script run](./29.png)
+
+  ![Screenshot showing the PowerShell logon script run](./30.png)
+
+  ![Screenshot showing the post-run desktop](./31.png)
+
+- Since this scenario is deploying the Azure Arc Data Controller and SQL Managed Instance, you will also notice additional newly deployed Azure resources in the resources group (at this point you should have **19 various Azure resources deployed**. The important ones to notice are:
+
+  - _Azure Arc-enabled Kubernetes cluster_ - Azure Arc-enabled data services deployed in directly connected are using this type of resource in order to deploy the data services [cluster extension](https://docs.microsoft.com/azure/azure-arc/kubernetes/conceptual-extensions) as well as for using Azure Arc [Custom locations](https://docs.microsoft.com/azure/azure-arc/kubernetes/conceptual-custom-locations).
+
+  - _Custom location_ - provides a way for tenant administrators to use their Azure Arc-enabled Kubernetes clusters as target locations for deploying Azure services instances.
+
+  - _Azure Arc Data Controller_ - The data controllers that are now deployed on the Kubernetes clusters.
+
+  - _Azure Arc-enabled SQL Managed Instance_ - The SQL Managed Instances that are now deployed on the Kubernetes clusters.
+
+    ![Screenshot showing additional Azure resources in the resource group](./32.png)
+
+- As part of the automation, Azure Data Studio is installed along with the _Azure Data CLI_, _Azure CLI_, _Azure Arc_ and the _PostgreSQL_ extensions. Using the Desktop shortcut created for you, open Azure Data Studio and click the Extensions settings to see the installed extensions.
+
+  ![Screenshot showing Azure Data Studio shortcut](./33.png)
+
+  ![Screenshot showing Azure Data Studio extensions](./34.png)
+
+- Additionally, the SQL Managed Instances connections will be configured automatically for you. As mentioned, the sample _AdventureWorks_ database was restored as part of the automation on the primary instance.
+
+  ![Screenshot showing Azure Data Studio SQL MI connection](./35.png)
+
+## Cluster extensions
+
+In this scenario, two Azure Arc-enabled Kubernetes cluster extensions were installed:
+
+- _azuremonitor-containers_ - The Azure Monitor Container Insights cluster extension. To learn more about it, you can check our Jumpstart ["Integrate Azure Monitor for Containers with GKE as an Azure Arc Connected Cluster using Kubernetes extensions](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/day2/gke/gke_monitor_extension/) scenario.
```

Comment:
Use the [AKS one](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/day2/aks/aks_monitor/), not GKE